Riot/Modules/Room/Members/Detail/RoomMemberDetailsViewController.xib

```diff
@@ -75,12 +75,12 @@
                             </constraints>
                         </view>
                         <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="7" translatesAutoresizingMaskIntoConstraints="NO" id="hSH-1t-jnC">
-                            <rect key="frame" x="0.0" y="112" width="375" height="221"/>
+                            <rect key="frame" x="0.0" y="112" width="375" height="164"/>
                             <subviews>
                                 <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="QHN-zr-wtC">
                                     <rect key="frame" x="0.0" y="0.0" width="375" height="50"/>
                                     <subviews>
-                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="252" text="Display name" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="i1p-3D-PhB" userLabel="Display name" customClass="CopyableLabel" customModule="Riot" customModuleProvider="target">
+                                        <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="252" text="Display name" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="i1p-3D-PhB" userLabel="Display name" customClass="CopyableLabel" customModule="Btchap" customModuleProvider="target">
```

Comment:
We may discuss about this change, at least I would expect Tchap reference (instead of Btchap)